extensions/amp-smartlinks/0.1/linkmate.js

```diff
@@ -163,12 +163,34 @@ export class Linkmate {
    * @private
    */
   getEditInfo_() {
+    const currentUrl = this.extractBaseUrl_(this.ampDoc_.getUrl());
+
     return dict({
       'name': this.rootNode_.title || null,
-      'url': this.ampDoc_.getUrl(),
+      'url': currentUrl,
     });
   }
 
+  /**
+   * Truncates AMP URLs to extract the base url for Narrativ Edit Tracking
+   * Purposes
+   * @param {?string} currentUrl The current URL of the AMP page
+   * @return {?string}
+   * @private
+   */
+  extractBaseUrl_(currentUrl) {
+    const splitPath = currentUrl.split('/');
+    let startIndex = 0;
+    while (startIndex < splitPath.length) {
+      if (splitPath[startIndex].startsWith('www.') ||
+        splitPath[startIndex].endsWith('.com')) {
+        return splitPath.slice(startIndex).join('/');
```

Comment:
We'll lose protocol here, right? I think we'll need that for urls to match from our normal jstag. ex: https://narrativ.looker.com/sql/jgq7bxwbjshzwk and https://narrativ.looker.com/sql/ftbhmxdjrh59vp, but not https://narrativ.looker.com/sql/m9zbfdngrvzcc3

Reply:
True. I'll grab protocol and prepend

Reply:
@Abalaxy @PhilWinchester check er out, what do you think?

Reply:
Added a conditional to check for `www-` as well so we don't unnecessarily split up the url